README.md

```diff
@@ -295,8 +295,43 @@ decay({ clock: Clock, value: Node, velocity: Node, deceleration: number} ): Node
 
 Convenience function to run a spring animation.
 
-```js
-decay({ clock: Clock, value: Node, velocity: Node, config: SpringConfig }): Node
+```tsx
+spring({ clock?: Clock, from?: Node, velocity?: number, config?: SpringConfig, to?: Node }): Node
+```
+
+### `toggle({ clock?: Clock, from?: Node, to?: Node, duration?: Node, easing?: EasingFunction }): Node`
+
+Convenience function to imperatively toggle animation.
+
+Example usage:
+
+```tsx
+toggle({
+  clock: Clock,
+  value: Node,
+  closeState: Node,
+  openState: Node,
+  easing: EasingFunction,
+  duration: number,
+  from: Node,
+  to: Node,
+})
+
+const handleToggleButtonPress = (shouldOpen: boolean) => {
+  shouldOpen ? open.setValue(1) : close.setValue(1)
+}
+
+useCode(
+  block([
+    toggle({
+      value: translationY,
+      openValue: open,
```

Comment:
Would it make sense to have only one value that is either one or zero?

Reply:
Yes, it would. However, I thought about readability. We can go for 1 value and after animation ends just set it State.END. What do you think?